app/graphql/connections/cursor_connection.rb

```diff
@@ -79,9 +79,10 @@ def load_nodes
     end
 
     def load_nodes_deprecated_order(limit, inverted)
+      Rails.logger.info "CursorConnection: using deprecated order [#{Current.user.email}]"
```

Comment:
pas complètement sûr mais je crois que ça doit être du json pour être visible dans kibana (parsable par logstash). @kleph ?

Reply:
en respectant les champs déjà définis ça pourrait être : 

```suggestion
      payload = {
        message: "CursorConnection: using deprecated order [#{Current.user.email}]",
        user_id: Current.user.id,
      }
      
      Rails.logger.info payload.to_json
```

Reply:
ah ok. merci